content/events/2022/07/2022-07-01-white-house-and-gsa-open-innovation-forum-building-equitable-partnerships.md

```diff
@@ -0,0 +1,71 @@
+
+At this event we will:
+
+1. Celebrate the role open innovation plays in meeting the federal government's goals for equitable participation to advance innovation, science, and technology.
+2. Recognize the achievement of open innovation practitioners across the federal government.
+3. Highlight partnerships and equitable forms of participation exemplified through prize competitions and citizen science projects.
+4. Explore the future of the federal government’s role in engaging with communities to create and lead innovation and scientific discovery activities.
+
+## Related Resources
+
+* Press release&mdash;[White House Office of Science & Technology Policy Releases Fiscal Year 2019-20 Federal Prize and Citizen Science Authority Report](https://www.whitehouse.gov/ostp/news-updates/2022/05/04/white-house-office-of-science-technology-policy-releasesfiscal-year-2019-20-federal-prize-and-citizen-science-authority-report/)
+* Report&mdash;[Implementation of Federal Prize and Citizen Science Authority: Fiscal Years 2019-20 (PDF, 10 MB, 917 pages, March 2022)](https://www.whitehouse.gov/wp-content/uploads/2022/05/05-2022-Implementation-of-Federal-Prize-and-Citizen-Science-Authority.pdf)
+
+  * Department of Transportation (DOT) - [Inclusive Design Challenge (begins on page 208 of the PDF)](https://www.whitehouse.gov/wp-content/uploads/2022/05/05-2022-Implementation-of-Federal-Prize-and-Citizen-Science-Authority.pdf#page=208&zoom=100,92,234);
+  * United States Agency for International Development (USAID) - [RISE Challenge (begins on page 542 of the PDF)](https://www.whitehouse.gov/wp-content/uploads/2022/05/05-2022-Implementation-of-Federal-Prize-and-Citizen-Science-Authority.pdf#page=542&zoom=100,92,750);
+  * National Oceanic and Atmospheric Administration (NOAA) - [Urban Heat Island Mapping Campaign (begins on page 608 of the PDF)](https://www.whitehouse.gov/wp-content/uploads/2022/05/05-2022-Implementation-of-Federal-Prize-and-Citizen-Science-Authority.pdf#page=608&zoom=100,92,494);
+  * Centers for Disease Control and Prevention (CDC) [COVID-19 Forecasting Hub (Epidemic Prediction Initiative) (begins on page 676 of the PDF)](https://www.whitehouse.gov/wp-content/uploads/2022/05/05-2022-Implementation-of-Federal-Prize-and-Citizen-Science-Authority.pdf#page=676&zoom=100,92,710);
+  * National Institute of Environmental Health Sciences (NIEHS) [Partnerships for Environmental Public Health (begins on page 837 of the PDF)](https://www.whitehouse.gov/wp-content/uploads/2022/05/05-2022-Implementation-of-Federal-Prize-and-Citizen-Science-Authority.pdf#page=837&zoom=100,92,800)
+
```

Comment:
@ToniBonittoGSA do we want to add a  line break here like we do for our other events or keep it in the main part of the copy because it is a forum? 

```suggestion
- - -
```